backend/models/listingModel.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const listingSchema = new Schema({
+  organisationName:{
+    type: String,
+    required:true
+  },
+  title: {
+    type: String,
+    required: true
+  },
+  description: {
+    type: String,
+    required:true
+  },
+  location: {
+    type: String,
+    required:true
+  },
+  neededByDate: {
+    type: Date,
+    required:true
+  }
+},{ timestamps: true}) 
```

Comment:
Nice! Agreed on timestamp as later on if we have time we can implement when a listing was created based on using the timestamp :)